src/Nox.Types/Types/Volume/Volume.cs

```diff
@@ -8,46 +9,130 @@ namespace Nox.Types;
 /// </summary>
 public class Volume : Measurement<Volume, VolumeUnit>
```

Comment:
This class should inherit from ValueObject according to the latest changes. If no other type is inheriting from Measurement, please delete the Measurement class.

Reply:
Class removed.